autoware.repos

```diff
@@ -45,6 +45,10 @@ repositories:
     type: git
     url: https://github.com/tier4/ndt_omp.git
     version: tier4/main
+  universe/external/cuda-api-wrappers:
```

Comment:
Don't to need some TODO comment here?
For example, `TODO(TIER IV): Remove after apt released.`

Reply:
```suggestion
  universe/external/cuda-api-wrappers: # TODO(TIER IV): Remove after apt released.
```